shopfloor_reception/services/reception.py

```diff
@@ -136,9 +136,16 @@ def _select_picking(self, picking):
             )
         return self._response_for_select_move(picking)
 
+    def _response_for_select_move_get_data(self, picking):
+        data = {}
+        if picking:
+            data = {"picking": self._data_for_stock_picking(picking, with_lines=True)}
+        return data
+
     def _response_for_select_move(self, picking, message=None):
-        self._assign_user_to_picking(picking)
-        data = {"picking": self._data_for_stock_picking(picking, with_lines=True)}
+        if picking:
+            self._assign_user_to_picking(picking)
+        data = self._response_for_select_move_get_data(picking)
```

Comment:
Why should the picking not set?

Reply:
After the rebase the implementation has changed on the reception service. The assign_user_to_picking is not used anymore. So not a question anymore.